<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UPrintForm</class>
 <widget class="QWidget" name="UPrintForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Вихідні документи  -&gt;  </string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/UImgLib.qrc">
    <normaloff>:/UtechLibImg/libImages/utechLibImg_icontexto-webdev-print-032x032.png</normaloff>:/UtechLibImg/libImages/utechLibImg_icontexto-webdev-print-032x032.png</iconset>
  </property>
  <layout class="QGridLayout">
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTextEdit" name="textEdit">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_printPreview">
       <property name="text">
        <string>Попередній перегляд</string>
       </property>
       <property name="icon">
        <iconset resource="../images/UImgLib.qrc">
         <normaloff>:/UtechLibImg/libImages/utechLibImg_icontexto-webdev-emoticon-smile-032x032.png</normaloff>:/UtechLibImg/libImages/utechLibImg_icontexto-webdev-emoticon-smile-032x032.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_exportInPdf">
       <property name="text">
        <string>Експорт в PDF</string>
       </property>
       <property name="icon">
        <iconset resource="../images/UImgLib.qrc">
         <normaloff>:/UtechLibImg/libImages/utechLibImg_pdf_document.png</normaloff>:/UtechLibImg/libImages/utechLibImg_pdf_document.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_print">
       <property name="minimumSize">
        <size>
         <width>83</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Друк</string>
       </property>
       <property name="icon">
        <iconset resource="../images/UImgLib.qrc">
         <normaloff>:/UtechLibImg/libImages/utechLibImg_icontexto-webdev-print-032x032.png</normaloff>:/UtechLibImg/libImages/utechLibImg_icontexto-webdev-print-032x032.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_print_on">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Друк на прінтері ...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="minimumSize">
        <size>
         <width>83</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Відміна</string>
       </property>
       <property name="icon">
        <iconset resource="../images/UImgLib.qrc">
         <normaloff>:/UtechLibImg/libImages/utechLibImg_icontexto-webdev-cancel-032x032.png</normaloff>:/UtechLibImg/libImages/utechLibImg_icontexto-webdev-cancel-032x032.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_print</tabstop>
  <tabstop>pushButton_print_on</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../images/UImgLib.qrc"/>
 </resources>
 <connections/>
</ui>
